webapp/app/model/form_data.py

```diff
@@ -90,7 +90,7 @@ class MandatoryFormData(BaseModel):
     person_a_blind: bool
     person_a_gehbeh: bool
     person_a_has_disability: str
-    person_a_requests_pauschbetrag: str
+    person_a_requests_pauschbetrag: Optional[str]
```

Comment:
I think this is still conditionally mandatory, right? I think this is what Julian meant with this comment: https://github.com/digitalservice4germany/steuerlotse/pull/516#discussion_r775396478

Reply:
Added the condition but can you double check it? :)